multicluster/apis/multicluster/v1alpha1/tunnelendpoint_types.go

```diff
@@ -0,0 +1,93 @@
+/*
+Copyright 2021 Antrea Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// TunnelEndpointSpec defines the desired state of TunnelEndpoint
+type TunnelEndpointSpec struct {
+	// Role (leader/member) shows the role of Gateway node if HA is enabled
+	Role string `json:"role,omitempty"`
+	// ClusterID of the member cluster.
+	ClusterID string `json:"clusterID,omitempty"`
+	//Hostname of Gateway node
+	Hostname string `json:"hostname,omitempty"`
```

Comment:
Why we need hostname? If it is just a name, call it "Name" or "GatewayName", even the implementation uses hostname.

Reply:
change it to Name